### PR TITLE
Feature/support nested captures

### PR DESCRIPTION
**Pull request by joshuafcole**
It's nice to be able to group captures by test suite. It may also be useful to Alon.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -33,7 +33,13 @@ module.exports = function init(opts) {
   var server = new Hapi.Server('0.0.0.0', opts.port, serverOpts);
 
   server.settings.app = opts;
-  server.settings.app.dbd = path.join(opts.cwd, (opts.dbd || 'run'));
+  var dbd = opts.dbd || 'run';
+  // This is awful, but normalize doesn't remove trailing slashes, and we can't compare against dbd directly in case it isn't normalized (e.g. /foo/../)
```
Comment (diffsky) [inline server.js]: this comment is probably not needed now

**Comment by diffsky** [comment]
I'm concerned this is going to break thumbnails in existing spook run reports. Given the idea is to view the images via the spook server output, I'm not sure of the value of the grouping.

**Comment by joshuafcole** [comment]
I'll refactor to prefix basename with `thumb.` then to match existing
output.

On Thu, Aug 20, 2015 at 1:03 PM, Matthew Hadley <notifications@github.com>
wrote:

> I'm concerned this is going to break thumbnails in existing spook run
> reports. Given the idea is to view the images via the spook server output,
> I'm not sure of the value of the grouping.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/creativelive/spook/pull/5#issuecomment-133155684>.
>



-- 
Screw the environment. Please print this email immediately. And then burn
it.

**Comment by diffsky** [comment]
Any change to the path (nested directory, filename change, etc)  is going to have this problem, as the files have already been written to disk, long ago.

**Comment by joshuafcole** [comment]
This will yield the same output for any single directory path, but now has
the capability of writing to subdirectories, which before would try to
write to a nonexistent directory and fail. This is just allowing writing to
subdirectories if the path specified when calling capture contains one,
rather than moving thumbnails around.

On Thu, Aug 20, 2015 at 2:51 PM, Matthew Hadley <notifications@github.com>
wrote:

> Any change to the path (nested directory, filename change, etc) is going
> to have this problem, as the files have already been written to disk, long
> ago.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/creativelive/spook/pull/5#issuecomment-133188484>.
>



-- 
Screw the environment. Please print this email immediately. And then burn
it.

**Comment by ryanjadhav** [comment]
your email signuature is great @joshuafcole 

```
-- 
Screw the environment. Please print this email immediately. And then burn
it.
```

**Comment by diffsky** [comment]
@joshuafcole turns out `path.isAbsolute` isn't on node `0.10` :( looks like it's on `0.12`.

**Comment by joshuafcole** [comment]
Does explain why I wasn't aware of it. I happen to be using node 0.12.5
locally, which is why it worked. I'll revert to the previous approach.

On Thu, Aug 20, 2015 at 4:05 PM, Matthew Hadley <notifications@github.com>
wrote:

> @joshuafcole <https://github.com/joshuafcole> turns out path.isAbsolute
> isn't on node 0.10 :( looks like it's on 0.12.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/creativelive/spook/pull/5#issuecomment-133207512>.
>



-- 
Screw the environment. Please print this email immediately. And then burn
it.

**Comment by diffsky** [comment]
Can you move the `dbd` change out of this PR and leave it for #4? Let's just make this about the thumbs. I've tested that change and it seems good. I'm just looking into lint issues

**Comment by joshuafcole** [comment]
Will do

**Comment by diffsky** [comment]
Thanks @joshuafcole - I'll merge and cleanup

**Comment by diffsky** [comment]
```
$ npm publish
+ spook@0.8.0
```

**Pull request merged**